src/Aeon.Acquisition/GetRepository.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using LibGit2Sharp;
+
+
+namespace Aeon.Acquisition
+{
+    [DefaultProperty(nameof(RepoPath))]
+    [Description("Returns a Repository object from the target repository root path. " +
+        "If no path is provided it will use the current working directory")]
+
+    public class GetRepository : Source<IRepository>
```

Comment:
I was worried that it might be confusing since "CreateRespository" could be infered to literally create a new repository (i.e. `git init`). I changed it to `CreateRepository`.